Validation/EcalClusters/src/EnergyScaleAnalyzer.cc

```diff
@@ -144,6 +144,7 @@ void EnergyScaleAnalyzer::analyze(const edm::Event &evt, const edm::EventSetup &
   Labels l;
   labelsForToken(hepMCLabel_, l);
 
+  [[clang::suppress]]
```

Comment:
`genEvent` is used later in the code. Clang static analyzer complains that  if `hepMC.isValid()` is not valid then `genEvent` is not used. ` [[clang::suppress]]` here instruct clang analyzer to ignore this warnings